.vib/mongodb/goss/mongodb.yaml

```diff
@@ -0,0 +1,10 @@
+file:
```

Comment:
You should check `/opt/bitnami/mongodb/conf/mongodb.conf` and some of its rendered content